plugin/src/main/groovy/com/btkelly/gnag/utils/ProjectHelper.groovy

```diff
@@ -20,18 +20,31 @@ import org.gradle.api.Project
 /**
  * Created by bobbake4 on 4/19/16.
  */
-class ReportHelper {
+class ProjectHelper {
 
-    private final Project project;
+    private final Project project
 
-    ReportHelper(Project project) {
+    ProjectHelper(Project project) {
         this.project = project
     }
 
-    // todo: this doesn't seem like it belongs in the ReportHelper. refactor?
-    public List<File> getAndroidSources() {
-        project.android.sourceSets.inject([]) {
-            dirs, sourceSet -> dirs + sourceSet.java.srcDirs
+    public Project getProject() {
+        return project
+    }
+
+    public boolean isAndroidProject() {
+        return project.hasProperty("android")
```

Comment:
Adapting something like this might be more robust? Both the plugin types extends `com.android.build.gradle.BasePlugin` I think.

```
       project.afterEvaluate(evaluatedProject -> {
           if (evaluatedProject.getPlugins().hasPlugin(AppPlugin.class)) {
               AppExtension android = (AppExtension) evaluatedProject.getExtensions().getByName("android");
               addAllTasksToProject(project, gnagPluginExtension, android.getApplicationVariants());
           } else if (evaluatedProject.getPlugins().hasPlugin(LibraryPlugin.class)) {
               LibraryExtension android = (LibraryExtension) evaluatedProject.getExtensions().getByName("android");
               addAllTasksToProject(project, gnagPluginExtension, android.getLibraryVariants());
           } else {
               // fixme: throw a loud exception here
           }
          });
```